### DISCUSSION TITLE
2023년 04월 25일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#46] [General]
# 2023년 04월 25일

LOG 네오의 면담 일정이 열렸다. 금요일 5시가 비어있길래 바로 신청했다. 딱 그 때가 가장 널널할 것 같았다. 면담 일정이 열림과 동시에 리뷰어->리뷰이 피드백이 와서 읽어봤다. 다행히(?) 좋은 말이 많아서 기분이 좋았다. 초반에 리뷰어한테 댓…

https://cyma-s.github.io/2023-04-25/

**Comment by be-student** [comment]
조급하면서 조급하지 않게 하죠
베로베로베로베로는 할 수 있을지도?

**Reply by Cyma-s** [reply]
역시 누누누누는 현명하네요?
